protoc-java/Dockerfile

```diff
@@ -1,26 +1,12 @@
 FROM ubuntu:16.04 as javalite_builder
 RUN apt-get update && \
     apt-get install -y bash patch curl patchelf
-ENV PROTOC_GEN_JAVALITE_VERSION=3.0.0
```

Comment:
java plugin is part of the protoc now, so we no longer need to install separately